src/components/Chart/plugins/chart.triggerLevel.js

```diff
@@ -135,59 +133,82 @@ const plugin = {
-        ctx.closePath();
-        ctx.fill();
-
-        ctx.lineWidth = 1;
-        ctx.strokeStyle = colors.gray50;
-        ctx.beginPath();
-        ctx.moveTo(4, 3);
-        ctx.lineTo(4, label.h - 3);
-        ctx.moveTo(8, 3);
-        ctx.lineTo(8, label.h - 3);
-        ctx.moveTo(12, 3);
-        ctx.lineTo(12, label.h - 3);
-        ctx.closePath();
-        ctx.stroke();
-
-        ctx.textAlign = 'right';
-        ctx.fillStyle = white;
-        ctx.fillText(formatY(triggerLevel), label.w - 18, label.h / 2 + 4);
+
+        function drawDashedLine() {
+            ctx.lineWidth = 0.5;
+            ctx.strokeStyle = color;
+            ctx.setLineDash([4, 5]);
+            ctx.beginPath();
+            ctx.moveTo(left, y - 1); // Moving it 1px up seems to center it on the label
+            ctx.lineTo(right, y - 1); // Moving it 1px up seems to center it on the label
+            ctx.closePath();
+            ctx.stroke();
+            ctx.setLineDash([]);
+        }
+
+        function drawHandle() {
+            ctx.fillStyle = nordicBlue;
+
+            ctx.translate(label.x, label.y);
+            ctx.beginPath();
+            ctx.moveTo(0, 2);
+            ctx.bezierCurveTo(0, 1, 1, 0, 2, 0);
+            const curveStart = label.w - 8;
+            ctx.lineTo(curveStart, 0);
+            ctx.bezierCurveTo(
+                label.w - 5,
+                0,
+                label.w,
+                label.h / 2,
+                label.w,
+                label.h / 2
+            );
+            ctx.bezierCurveTo(
+                label.w,
+                label.h / 2,
+                label.w - 5,
+                label.h,
+                curveStart,
+                label.h
+            );
+            ctx.lineTo(2, label.h);
+            ctx.bezierCurveTo(1, label.h, 0, label.h - 1, 0, label.h - 2);
+            ctx.closePath();
+            ctx.fill();
+
+            ctx.lineWidth = 1;
+            ctx.strokeStyle = colors.gray50;
+            ctx.beginPath();
+            ctx.closePath();
+            ctx.stroke();
+        }
+
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        function drawHandleLines() {
+            ctx.lineWidth = 1;
+            ctx.strokeStyle = colors.gray50;
+            ctx.beginPath();
+            ctx.moveTo(4, 3);
+            ctx.lineTo(4, label.h - 3);
+            ctx.moveTo(8, 3);
+            ctx.lineTo(8, label.h - 3);
+            ctx.moveTo(12, 3);
+            ctx.lineTo(12, label.h - 3);
+            ctx.closePath();
+            ctx.stroke();
+        }
+
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        function drawText() {
+            ctx.textAlign = 'right';
+            ctx.fillStyle = white;
+            ctx.fillText(formatY(triggerLevel), label.w - 18, label.h / 2 + 4);
+        }
+
+        drawDashedLine();
+        drawHandle();
+        // drawHandleLines();
+        // drawText(ctx);
```

Comment:
This plugin can be well cleaned up, since it does not draw the text anymore, so it will not need to rely on the _label_ values.